### DISCUSSION TITLE
how do you update jsonb object from node application

### DISCUSSION THREAD

**Discussion by lienista** [supabase/supabase#3966] [Questions]
I have a jsonb data type called `metadata` and i'm trying to perform an update

```
const { data_timezone, err_timezone } = await supabase
      .from("users")
      .update({
        metadata: JSON.stringify(tz)
      })
      .match({ email: user.email });
```

This doesn't work, was wondering how to do this.

**Comment by ltanady** [comment]
Can you let us know what is the output for `data_timezone` and `err_timezone`?

**Comment by dshukertjr** [comment]
Hi @lienista!

You might be able to get some hints if you could log the contents of your error object. 

```typescript
const { data: data_timezone, error: err_timezone } = await supabase
  .from("users")
  .update({
     metadata: JSON.stringify(tz)
   })
  .match({ email: user.email });

console.log('data_timezone', data_timezone)
console.log('err_timezone', err_timezone)
```

Also, it might help us to know what is inside `tz` in order to guide you. Could you share the output of the following code as well?

```typescript
console.log('tz', tz)
```
